packages/runtime/src/widgets/ImageCropper.tsx

```diff
@@ -38,15 +38,12 @@ export const ImageCropper: React.FC<ImageCropperProps> = (props) => {
   });
 
   const onCroppedAction = useEnsembleAction(onCropped);
-  const onCroppedActionCallback = useCallback(
-    (file: unknown) => {
-      if (!onCroppedAction) {
-        return;
-      }
-      onCroppedAction.callback({ file });
-    },
-    [onCroppedAction],
-  );
+  const onCroppedActionCallback = useCallback((file: unknown) => {
+    if (!onCroppedAction) {
+      return;
+    }
+    onCroppedAction.callback({ file });
```

Comment:
So this is an example where you do need the action in the dependency, or at the last the callback. If the callback hook changes any of the dependencies this does need to get recomputed.